### DISCUSSION TITLE
Order Returns endpoint

### DISCUSSION THREAD

**Discussion by amiddles** [amzn/selling-partner-api-models#583] [Ideas]
Please consider adding a new API section for managing customer returns.
functions should include
- locate return requests
- update a return with the correct RMA, message, label, etc.
- complete a return including being able to specify any restocking fees, attach any supporting images, etc.

Furthermore, I'd recommend changing the functionality of the automated returns system to allow merchants using the API to have some time to respond to any return request prior to amazon authorizing it so that a correct RMA number can be provided as well as any mitigating efforts be made to resolve the issue before a return is approved.

**Comment by maximehinnekens** [comment]
Yes we need this, ive been asking support but I guess its not coming soon. Very strange to me that this is not existing in the api of the biggest marketplace in the world
